### PR TITLE
Feature/pb 20 footer

### PR DESCRIPTION
**Pull request by GabrielCFurlan**
image

### CODE REVIEW

File: src/components/Footer/Footer.style.js [hunk 1]
```diff
@@ -1,5 +1,56 @@
...
-export const FooterWrapper = styled.footer``
+export const FooterWrapper = styled.footer`
+  display: flex;
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  height: 120px;
+  background-color: ${({ theme }) => theme.colors.secondary};
+`
+export const Description = styled.div``
```
Comment (Jejugo) [inline src/components/Footer/Footer.style.js]: Remove empty styled-component styles

File: src/components/Footer/Footer.style.js [hunk 2]
```diff
@@ -1,5 +1,56 @@
...
+
+export const LinkTitle = styled.div``
+
+export const LinkPages = styled(Link)`
+  display: flex;
+`
+
+export const BlogLinks = styled.div``
+
+export const CategorieLinks = styled.div``
```
Comment (Jejugo) [inline src/components/Footer/Footer.style.js]: again typo on category please study english more often

**Review by Jejugo — COMMENTED** [review]
Almost there

File: src/components/Footer/Footer.style.js [hunk 3]
```diff
@@ -1,5 +1,86 @@
...
+  align-items: center;
+  margin: 0 auto;
+  padding: 0 20px;
+`
+
+export const LinkTitle = styled.div`
+  font-weight: ${({ theme }) => theme.fontWeights.bold};
+`
+
+export const LinkPages = styled(Link)`
```
Comment (Jejugo) [inline src/components/Footer/Footer.style.js]: You can re-use the Flex component Nicolas created

File: src/components/Footer/Footer.style.js [hunk 4]
```diff
@@ -1,5 +1,86 @@
...
+export const StyledImage = styled(Image)`
+  border-radius: 50%;
+`
+
+export const Title = styled.div`
+  font-weight: ${({ theme }) => theme.fontWeights.bold};
+  font-size: ${({ theme }) => theme.fontSizes.medium};
+`
+
+export const SocialMedia = styled(Link)`
```
Comment (Jejugo) [inline src/components/Footer/Footer.style.js]: You can re-use the Flex component Nicolas created

File: src/components/Footer/Footer.style.js [hunk 5]
```diff
@@ -1,5 +1,86 @@
...
+
+export const LinkTitle = styled.div`
+  font-weight: ${({ theme }) => theme.fontWeights.bold};
+`
+
+export const LinkPages = styled(Link)`
+  display: flex;
+`
+
+export const BlogLinks = styled.div`
```
Comment (Jejugo) [inline src/components/Footer/Footer.style.js]: You can re-use the Flex component Nicolas created

File: src/components/Footer/Footer.style.js [hunk 6]
```diff
@@ -1,5 +1,86 @@
...
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+`
+
+export const CategorieLinks = styled.div`
```
Comment (Jejugo) [inline src/components/Footer/Footer.style.js]: You can re-use the Flex component Nicolas created

File: src/components/Footer/Footer.style.js [hunk 7]
```diff
@@ -1,5 +1,86 @@
...
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+`
+
+export const SocialLinks = styled.div`
```
Comment (Jejugo) [inline src/components/Footer/Footer.style.js]: You can re-use the Flex component Nicolas created

File: src/components/Footer/Footer.style.js [hunk 8]
```diff
@@ -1,5 +1,86 @@
...
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+`
+
+export const SocialIcons = styled.div`
```
Comment (Jejugo) [inline src/components/Footer/Footer.style.js]: You can re-use the Flex component Nicolas created

File: src/components/Footer/Footer.style.js (hunk 1 above)
Comment (GabrielCFurlan) [reply src/components/Footer/Footer.style.js]: Done

File: src/components/Footer/Footer.style.js (hunk 2 above)
Comment (GabrielCFurlan) [reply src/components/Footer/Footer.style.js]: Done

File: src/components/Footer/Footer.style.js (hunk 3 above)
Comment (GabrielCFurlan) [reply src/components/Footer/Footer.style.js]: Done

File: src/components/Footer/Footer.style.js (hunk 4 above)
Comment (GabrielCFurlan) [reply src/components/Footer/Footer.style.js]: Done

File: src/components/Footer/Footer.style.js (hunk 5 above)
Comment (GabrielCFurlan) [reply src/components/Footer/Footer.style.js]: Done

File: src/components/Footer/Footer.style.js (hunk 6 above)
Comment (GabrielCFurlan) [reply src/components/Footer/Footer.style.js]: Done

File: src/components/Footer/Footer.style.js (hunk 7 above)
Comment (GabrielCFurlan) [reply src/components/Footer/Footer.style.js]: Done

File: src/components/Footer/Footer.style.js (hunk 8 above)
Comment (GabrielCFurlan) [reply src/components/Footer/Footer.style.js]: Done

**Review by Jejugo — APPROVED** [review]
Good job!

**Review by Cacatuaa — APPROVED** [review]

**Pull request merged**